libs/scripts/delayed.js

```diff
@@ -1,7 +1,54 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* globals */
 // eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './scripts.js';
+import { loadScript, sampleRUM } from './scripts.js';
```

Comment:
If you are going to import loadScript, it should be from utils.js not scripts.js

Reply:
hah, can't believe I missed this.  thanks again for picking that up.  I made the change to pull it from utils.js